<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ddcutil - mainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="views_stackedWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>42</y>
      <width>618</width>
      <height>206</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_moninfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <widget class="QPlainTextEdit" name="moninfoPlainText">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>6</y>
        <width>574</width>
        <height>191</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>574</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_capabilities">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <widget class="QPlainTextEdit" name="capabilities_plainText">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>6</y>
        <width>574</width>
        <height>231</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>574</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="centerOnScroll">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_vcp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <widget class="QListWidget" name="vcp_listWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>1</y>
        <width>771</width>
        <height>251</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_table_item">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-9</y>
        <width>751</width>
        <height>251</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>581</width>
        <height>0</height>
       </size>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <column>
       <property name="text">
        <string>Code</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RW</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="page_table_view">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="vcp_tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>561</width>
          <height>0</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_list_view">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>LM Mono 9</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <widget class="QListView" name="vcp_listView">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>6</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>268</x>
        <y>6</y>
        <width>342</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Header line for page_list_view containing vcp_listView</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionMonitor_Summary"/>
    <addaction name="actionCapabilities"/>
    <addaction name="actionFeaturesListView"/>
    <addaction name="actionFeatures_TableView"/>
   </widget>
   <widget class="QMenu" name="menuDisplays">
    <property name="title">
     <string>Disp&amp;lays</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Optio&amp;ns</string>
    </property>
    <addaction name="actionFeature_Selection_Dialog"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuDisplays"/>
   <addaction name="menuHelp"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About ddcui</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionMonitor_Summary">
   <property name="text">
    <string>&amp;Monitor Summary</string>
   </property>
  </action>
  <action name="actionCapabilities">
   <property name="text">
    <string>&amp;Capabilities</string>
   </property>
  </action>
  <action name="actionFeaturesListView">
   <property name="text">
    <string>&amp;Features - ListView</string>
   </property>
  </action>
  <action name="actionFeature_Selection">
   <property name="text">
    <string>Feature Selection</string>
   </property>
  </action>
  <action name="actionShow_Unsupported_Features">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Unsupported Features</string>
   </property>
  </action>
  <action name="actionKnown">
   <property name="text">
    <string>Known</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="text">
    <string>Scan</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionProfile">
   <property name="text">
    <string>Profile</string>
   </property>
  </action>
  <action name="actionManufacturer">
   <property name="text">
    <string>Manufacturer</string>
   </property>
  </action>
  <action name="actionFeature_Selection_Dialog">
   <property name="text">
    <string>&amp;Feature Selection</string>
   </property>
  </action>
  <action name="actionFeatures_TableView">
   <property name="text">
    <string>F&amp;eatures - TableView</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
